lib/eventbrite/api/model/event.rb

```diff
@@ -6,6 +6,10 @@ def model_route
           'events'
         end
 
+        def delete(id)
```

Comment:
Since we are using delete only for Event right now and haven't tested with the other models, let us have the delete functionality here itself, instead of in Base model.

I know it is a good idea to have it in the Base model, but let us move it there when we need delete for another mdoel

Reply:
Oh ok sure